### PR TITLE
ARUHA-1340 Remove topology logging on cursors commit

### PR DESCRIPTION
**Pull request by antban**
[ARUHA-1340: Remove a lot of logging from cursors commit]

## Description
Right now on cursor commit nakadi logs full topology. This is not needed at all. Removed.

### CODE REVIEW

File: src/main/java/org/zalando/nakadi/service/subscription/zk/NewZkSubscriptionClient.java [hunk 1]
```diff
@@ -111,9 +113,7 @@ private Topology readTopology() throws NakadiRuntimeException,
 
     private Topology parseTopology(final byte[] data) {
         try {
-            final Topology result = objectMapper.readValue(data, Topology.class);
-            getLog().info("Topology is {}", result);
```
Comment (v-stepanov) [inline src/main/java/org/zalando/nakadi/service/subscription/zk/NewZkSubscriptionClient.java]: Topology is Topology

**Comment by v-stepanov** [comment]
👍

File: src/main/java/org/zalando/nakadi/service/subscription/zk/NewZkSubscriptionClient.java (hunk 1 above)
Comment (antban) [reply src/main/java/org/zalando/nakadi/service/subscription/zk/NewZkSubscriptionClient.java]: @v-stepanov It's strange, but nothing will change :)

**Comment by antban** [comment]
:+1:

**Comment by antban** [comment]
deploy validation please

**Pull request merged**